1-js/01-getting-started/2-manuals-specifications/article.md

```diff
@@ -1,42 +1,42 @@
+[ECMA-262 विनिर्देशन] (https://www.ecma-international.org/publications/standards/Ecma-262.htm) में {जावास्क्रिप्ट} के बारे में सबसे अधिक गहन, विस्तृत और औपचारिक जानकारी है। यह भाषा को परिभाषित करता है।
 
-But being that formalized, it's difficult to understand at first. So if you need the most trustworthy source of information about the language details, the specification is the right place. But it's not for everyday use.
+लेकिन औपचारिक होने के नाते, यह पहली बार में समझना मुश्किल है। इसलिए यदि आपको भाषा के विवरण के बारे में सबसे भरोसेमंद स्रोत की आवश्यकता है, तो विनिर्देश सही जगह है। लेकिन यह रोजमर्रा के उपयोग के लिए नहीं है।
 
-A new specification version is released every year. In-between these releases, the latest specification draft is at <https://tc39.es/ecma262/>.
+हर साल एक नया विनिर्देश संस्करण जारी किया जाता है। इन रिलीज़ों के बीच, नवीनतम विनिर्देश ड्राफ्ट <https://tc39.es/ecma262/> पर है।
 
-To read about new bleeding-edge features, including those that are "almost standard" (so-called "stage 3"), see proposals at <https://github.com/tc39/proposals>.
+नई ब्लीडिंग-एज विशेषताओं के बारे में पढ़ने के लिए, जिनमें "लगभग मानक" (तथाकथित "स्टेज 3") शामिल हैं, <https://github.com/tc39/proposals> पर प्रस्ताव देखें।
 
-Also, if you're in developing for the browser, then there are other specs covered in the [second part](info:browser-environment) of the tutorial.
+इसके अलावा, यदि आप ब्राउज़र के लिए विकसित कर रहे हैं, तो ट्यूटोरियल के [दूसरे भाग] (जानकारी: ब्राउज़र-पर्यावरण) में शामिल अन्य चश्मा हैं।
 
-## Manuals
+## मैनुअल
 
-- **MDN (Mozilla) JavaScript Reference** is a manual with examples and other information. It's great to get in-depth information about individual language functions, methods etc.
+- **एमडीएन (मोज़िला) जावास्क्रिप्ट संदर्भ** उदाहरण और अन्य जानकारी के साथ एक मैनुअल है। व्यक्तिगत भाषा के कार्यों, विधियों आदि के बारे में गहराई से जानकारी प्राप्त करना बहुत अच्छा है।
 
-    One can find it at <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference>.
+    कोई इसे <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference> पर पा सकता है।
 
-    Although, it's often best to use an internet search instead. Just use "MDN [term]" in the query, e.g. <https://google.com/search?q=MDN+parseInt> to search for `parseInt` function.
+    हालांकि, इसके बजाय अक्सर इंटरनेट खोज का उपयोग करना सबसे अच्छा है। बस क्वेरी में "एमडीएन [टर्म]" का उपयोग करें, उदा। {`parseInt`} फ़ंक्शन की खोज करने के लिए <https://google.com/search?q=MDN parseInt>।
 
 
-- **MSDN** – Microsoft manual with a lot of information, including JavaScript (often referred to as JScript). If one needs something specific to Internet Explorer, better go there: <http://msdn.microsoft.com/>.
+- **MSDN** – Microsoft बहुत सारी जानकारी के साथ मैनुअल करता है, जिसमें जावास्क्रिप्ट (अक्सर JScript के रूप में संदर्भित) शामिल है। अगर किसी को इंटरनेट एक्सप्लोरर के लिए कुछ विशिष्ट चाहिए, तो बेहतर तरीके से वहां जाएं: <http://msdn.microsoft.com/>।
```

Comment:
This sentence can be improved. The first sentence doesn't convey the meaning well.